src/casp/services/db/migrations/versions/t_2024-03-13_14-58-14_non_unique_model_path.py

```diff
@@ -0,0 +1,27 @@
+"""non unique model path
+
+Revision ID: a2e0ae902a06
+Revises: 2633d266e26d
+Create Date: 2024-03-13 14:58:14.174652
+
+"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a2e0ae902a06"
+down_revision = "2633d266e26d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Same here.  We can probably get rid of these comments.  Unless we want to remember which changes were autogenerated?